README.md

```diff
@@ -130,6 +131,9 @@ There isn't a current roadmap for this plugin, but it is in active development.
 
 If you have a feature request, please add it as an [issue](https://github.com/intelsdi-x/snap-plugin-publisher-graphite/issues/new) and/or submit a [pull request](https://github.com/intelsdi-x/snap-plugin-publisher-graphite/pulls).
 
+### Known issues
+Graphite plugin does not check whether on address to which it publishes griphite is running or not. If exists any service on specific address and port Graphite Publisher should not crash.
```

Comment:
That makes more sense now. You should add that description to the GitHub issue you create to address this.